ui/src/app/shared/models.ts

```diff
@@ -639,6 +646,7 @@ export interface Project {
     kind?: string;
     metadata: models.ObjectMeta;
     spec: ProjectSpec;
+    status: AppProjectStatus;
```

Comment:
Remove obsolete `jwtTokens` field from  [ProjectRole](https://github.com/mayzhang2000/argo-cd/blob/69ee03fa1edd993253756d080d65ea35e063d32f/ui/src/app/shared/models.ts#L600) interface to make sure we are not using it anymore.